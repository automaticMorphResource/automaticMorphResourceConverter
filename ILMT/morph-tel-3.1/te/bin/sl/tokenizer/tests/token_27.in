కాకతీయ రాజులవద్ద సేనానులుగా పనిచేసిన పద్మనాయక వంశానికి చెందిన భేతాళ నాయకుడి సంతతివారు ఆ తరువాతి కాలంలో దేవరకొండ రాజ్యాన్ని ఏర్పాటు చేసుకున్నట్లుగా చారిత్రక కథనాల ద్వారా తెలుస్తోంది . 
