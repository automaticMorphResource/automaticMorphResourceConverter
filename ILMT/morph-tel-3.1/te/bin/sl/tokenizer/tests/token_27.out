<Sentence id="1">
1	కాకతీయ	unk
2	రాజులవద్ద	unk
3	సేనానులుగా	unk
4	పనిచేసిన	unk
5	పద్మనాయక	unk
6	వంశానికి	unk
7	చెందిన	unk
8	భేతాళ	unk
9	నాయకుడి	unk
10	సంతతివారు	unk
11	ఆ	unk
12	తరువాతి	unk
13	కాలంలో	unk
14	దేవరకొండ	unk
15	రాజ్యాన్ని	unk
16	ఏర్పాటు	unk
17	చేసుకున్నట్లుగా	unk
18	చారిత్రక	unk
19	కథనాల	unk
20	ద్వారా	unk
21	తెలుస్తోంది	unk
22	.	unk
</Sentence>

